.travis.yml

```diff
@@ -14,20 +14,23 @@ before_install:
 - sudo apt-get -qq update
 - sudo apt-get install -y libgdal-dev
 - sudo apt-get install graphviz
+- sudo apt-get install proj-bin
+- sudo apt-get install gcc
+- sudo apt-get install libproj-dev
 
 install:
 - pip install -r requirements-dev.txt --upgrade
 - python install_all.py
 - pip install pandas==0.23.4  # This is temporal solution
 
 script:
-- pylint core/eolearn/core/*.py
-- pylint coregistration/eolearn/coregistration/*.py
-- pylint features/eolearn/features/*.py
-- pylint geometry/eolearn/geometry/*.py
-- pylint io/eolearn/io/*.py
-- pylint mask/eolearn/mask/*.py
-- pylint ml_tools/eolearn/ml_tools/*.py
+- if [[ $TRAVIS_PYTHON_VERSION == '3.6' ]]; then pylint core/eolearn/core/*.py ; fi
+- if [[ $TRAVIS_PYTHON_VERSION == '3.6' ]]; then pylint coregistration/eolearn/coregistration/*.py ; fi
+- if [[ $TRAVIS_PYTHON_VERSION == '3.6' ]]; then pylint features/eolearn/features/*.py ; fi
+- if [[ $TRAVIS_PYTHON_VERSION == '3.6' ]]; then pylint geometry/eolearn/geometry/*.py ; fi
+- if [[ $TRAVIS_PYTHON_VERSION == '3.6' ]]; then pylint io/eolearn/io/*.py ; fi
+- if [[ $TRAVIS_PYTHON_VERSION == '3.6' ]]; then pylint mask/eolearn/mask/*.py ; fi
+- if [[ $TRAVIS_PYTHON_VERSION == '3.6' ]]; then pylint ml_tools/eolearn/ml_tools/*.py ; fi
```

Comment:
I'm not very good with shell commands but I think something like this should work as well:
```bash
- if [[ $TRAVIS_PYTHON_VERSION == '3.6' ]]; then
  pylint core/eolearn/core/*.py
  pylint coregistration/eolearn/coregistration/*.py
  pylint features/eolearn/features/*.py
  pylint geometry/eolearn/geometry/*.py
  pylint io/eolearn/io/*.py
  pylint mask/eolearn/mask/*.py
  pylint ml_tools/eolearn/ml_tools/*.py
fi
```

Reply:
added ';' at the end of lines
inserted indents